packages/react/src/components/Form/index.ts

```diff
@@ -5,6 +5,4 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import Form from './Form';
-export default Form;
-export { Form };
```

Comment:
I think we'll need to keep the named export in the event that someone is relying on it (unlikely but possible)

Reply:
Should be fixed!